.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md

```diff
@@ -15,9 +15,16 @@ If you are reporting a bug, we would like to know:
 * Technical details like the version of Mu you're using, your OS version and
   other aspects of the context in which Mu was running. 
 
-Please remember to attach a **copy of the full log files for Mu**. You can get
+<!-- Please remember to add a **copy of the full log files for Mu** below. You can get
 the logs by clicking on the cog icon in the bottom right of the editor window.
 Click on the logs and use CTRL-A to select all, then CTRL-C to copy and CTRL-V
-to paste the contents into the issue.
+to paste the contents into the issue. -->
```

Comment:
If think perhaps if this request stays as a comment, instead of text that needs to be manually removed (like the other bullet points above), is probably more likely people might miss it?
To be fair, is just a guess, but i think I'd prefer to leave it out of the comment.

```suggestion
Please remember to add a **copy of the full log files for Mu** below.

<!--
You can get the logs by clicking on the cog icon in the bottom right of the
editor window.
Click on the logs and use CTRL-A to select all, then CTRL-C to copy and CTRL-V
to paste the contents below where it says:
"Please replace this line with your logs..."
-->
```